package.json

```diff
@@ -20,6 +20,7 @@
     "affected:lint": "nx affected:lint",
     "affected:dep-graph": "nx affected:dep-graph",
     "affected:codegen": "nx affected --target=codegen",
+    "affected:schemas": "nx affected --target=init-schema --parallel --maxParallel=5 --exclude=$(grep EXCLUDES_SCHEMAS .env | cut -d '=' -f2)",
```

Comment:
Let's say I'm only working on api/web project, I can do `EXCLUDES_SCHEMAS=air-discount-scheme-api,gjafakort-api,skilavottord-ws`. It saves a few seconds